### DISCUSSION TITLE
Συμπληρωματική ΟΣΣ4

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#67] [Announcements]
~~Σάββατο 18 Μαρτίου 6μμ~~ #68

αν υπάρχουν 2-3 παρουσιάσεις

**Comment by vkaragia** [comment]
Καλησπέρα σας, θα ήθελα να παρουσιάσω και εγώ ένα άρθρο για το Μέρος Δ της ΓΕ4 στην ΣΔΥ 60. Έχω μιλήσει και με τον Γρηγόρη όπου δεν μπόρεσε να παρουσιάσει στην προηγούμενη συμπληρωματική ΟΣΣ και σας προτείνω , να δεν έχετε αντίρρηση, να μην γίνει η συμπληρωματική αυτό το Σάββατο 18/3 αλλά να παρουσιάσουμε τα άρθρα μας στην κανονική ΟΣΣ στις 26/3 το πρωί. Λόγω και της αλλαγής  της ημερομηνίας υποβολής της εργασίας (29/3 αντί για 22/3) , θα είμαστε και τυπικά εντός πλαισίου.

**Comment by ghost** [comment]
Τελικά πότε θα γίνει η ΟΣΣ;

**Reply by afoscolou** [reply]
Είναι προγραμματισμένη επίσημη ΟΣΣ Κυριακή 26/3 στις 11:00.  Προσοχή στην αλλαγή ώρας που γίνεται τα ξημερώματα της Κυριακής!
